module/applications/advancement/ability-score-improvement-flow.mjs

```diff
@@ -87,10 +87,19 @@ export default class AbilityScoreImprovementFlow extends AdvancementFlow {
       return obj;
     }, {});
 
+    let recommendation = this.advancement.isEpicBoon ? fromUuidSync(this.advancement.configuration.recommendation)
+      : null;
+    if ( recommendation ) recommendation = {
+      ...recommendation,
+      checked: this.feat?.uuid === recommendation.uuid,
+      locked: this.feat && (this.feat.uuid !== recommendation.uuid),
+      uuid: recommendation.uuid
+    };
```

Comment:
Prefer to pluck only the properties we need, rather than getting everything with the spread operator.

```suggestion
    if ( recommendation ) {
      const { img, name, uuid } = recommendation;
      recommendation = {
        img, name, uuid,
        checked: this.feat?.uuid === uuid,
        locked: this.feat && (this.feat.uuid !== uuid)
      };
    }
```